api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceService.java

```diff
@@ -78,4 +80,8 @@ UserRecentWorkspace updateRecentWorkspaces(
   UserRecentWorkspace updateRecentWorkspaces(Workspace workspace);
 
   boolean maybeDeleteRecentWorkspace(long workspaceId);
+
+  String getRegisteredUserDomainEmail();
+
+  void setWorkbenchConfigProvider(Provider<WorkbenchConfig> configProvider);
```

Comment:
We actually found a new way to do this that doesn't require exposing a testing only method.


See `FreeTierBillingServiceTest.java` for an example. `Provider<User` within `WorkspaceControllerTest.java` also follows this method.

Reply:
I'll file a ticket to clean this up across our test files so there's no more outdated examples.

Reply:
Ooh, sweet